client/src/components/RelatedObjectNoteModal.js

```diff
@@ -106,26 +115,63 @@ const RelatedObjectNoteModal = ({
                     }
                     vertical
                   />
+                  <RelatedObjectsTable
+                    relatedObjects={relatedObjects}
+                    currentObject={edit ? undefined : currentObject}
+                    onSelect={handleRelatedObjectSelect}
+                    showDelete
+                    onDelete={handleRelatedObjectDelete}
+                  />
                 </div>
               </Modal.Body>
               <Modal.Footer>
                 <Button className="pull-left" onClick={close}>
                   Cancel
                 </Button>
-                <Button
-                  onClick={submitForm}
-                  bsStyle="primary"
-                  disabled={isSubmitting || !isValid}
-                >
-                  Save
-                </Button>
+                {_isEmpty(relatedObjects) ? (
+                  <ConfirmDelete
+                    onConfirmDelete={() => onDelete(note.uuid)}
```

Comment:
`onDelete` might be undefined (see https://github.com/NCI-Agency/anet/blob/GH-2954/client/src/components/RelatedObjectNotes.js#L183-L193 ); what should happen in that case?

Reply:
I've changed it to only display the delete button if onDelete is defined.